PolyEngine/Engine/Src/Movement/MovementSystem.cpp

```diff
@@ -15,7 +15,7 @@ void MovementSystem::MovementUpdatePhase(World* world)
 		FreeFloatMovementComponent* freeFloatMovementCmp = std::get<FreeFloatMovementComponent*>(freeFloatTuple);
 		EntityTransform& trans = freeFloatMovementCmp->GetTransform();
 
-		int wheelDelta = inputCmp->GetWheelPosDelta().Y;
+		int wheelDelta = 10 * inputCmp->GetWheelPosDelta().Y;
```

Comment:
Again, magic number change that is engine wide. Move to SandboxRendering or provide option to override wheel multiplier.